docs/source/field_analysis.rst

```diff
@@ -1689,45 +1689,99 @@ checks on the metadata. See :ref:`Operating-on-the-field-constructs-data`
 for more details. *(This will be made easier in a future release with
 a new function for combining such field constructs.)*
 
+.. _Bounds:
 
-.. warning:: Care must be taken when combining a field construct with a
-          `numpy` array or a `Data` instance, due to the ways in which
-          both of these objects allow themselves to be combined with
-          other types:
-
-	  * If the field construct is on the left hand side (LHS) of
-            the operation then, as expected, a field construct is
-            returned whose data is the combination of the original
-            field construct's data and the `numpy` array or `Data`
-            instance on the right hand side (RHS).
+.. rubric:: Bounds
+	    
+For binary operations involving constructs that have bounds, the
+result of binary operation will, by default, only have bounds if both
+operands have bounds; and the bounds of the result will be the result
+of the same binary operation on bounds objects. This behaviour may
+modified by the `cf.combine_bounds_with_coordinates` function.
 
-	  * If, however, the field construct is on the RHS then a
-            `numpy` array or `Data` instance (which ever type is on
-            the LHS) is returned, containing the same data as in the
-            first case.
+.. code-block:: python
+   :caption: *Demonstrate how bounds are treated in binary
+             operations.*
+
+   >>> x = q.dimension_coordinate('X')
+   >>> x.dump()
+   Dimension coordinate: longitude
+       standard_name = 'longitude'
+       units = 'degrees_east'
+       Data(longitude(8)) = [22.5, ..., 337.5] degrees_east
+       Bounds:units = 'degrees_east'
+       Bounds:Data(longitude(8), 2) = [[0.0, ..., 360.0]]
+   >>> (x + x).dump()
+   Dimension coordinate: longitude
+       standard_name = 'longitude'
+       units = 'degrees_east'
+       Data(8) = [45.0, ..., 675.0] degrees_east
+       Bounds:units = 'degrees_east'
+       Bounds:Data(8, 2) = [[0.0, ..., 720.0]] degrees_east
+   >>> (x + 50).dump()
+   Dimension coordinate: longitude
+       standard_name = 'longitude'
+       units = 'degrees_east'
+       Data(8) = [72.5, ..., 387.5] degrees_east   
+   >>> old = cf.combine_bounds_with_coordinates('OR')
+   >>> (x + 50).dump()
+   Dimension coordinate: longitude
+       standard_name = 'longitude'
+       units = 'degrees_east'
+       Data(8) = [72.5, ..., 387.5] degrees_east
+       Bounds:units = 'degrees_east'
+       Bounds:Data(8, 2) = [[50.0, ..., 410.0]] degrees_east
+   >>> x2 = x.copy()
+   >>> x2.del_bounds()
+   <CF Bounds: longitude(8, 2) degrees_east>
+   >>> (x2 + x).dump()
+   Dimension coordinate: longitude
+       standard_name = 'longitude'
+       units = 'degrees_east'
+       Data(8) = [45.0, ..., 675.0] degrees_east
+       Bounds:units = 'degrees_east'
+       Bounds:Data(8, 2) = [[22.5, ..., 697.5]] degrees_east
```

Comment:
Is this one correct? I was expecting, since `x2` has no bounds after the deletion and we're on `OR` mode so the bounds of `x` should be dumped, with `x` not having been changed, ``Bounds:Data(longitude(8), 2) = [[0.0, ..., 360.0]]``. Have I missed something?

Reply:
I think this is OK - in `OR` mode if both operands support bounds but only one actually has bounds, then the result will have bounds. So in the case of  `x2 + x`, `x2` doent have bounds, `x` does, so the result has bounds of `x2 + x.bounds`

Reply:
Aha, of course, thanks for talking it through! So it's an `op(x, y.bounds)` case. Looking back at the docstring you've written that's made quite clear, and originally I understood it fine, I just forgot the data operand part when faced with some actual numbers...